pkg/apis/controller/common/v1beta1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
Did you run `make generate` to update these files ?
What version of Go are you using ?

Reply:
> Did you run make generate to update these files ?

I didn't run `make generate`.

> What version of Go are you using ?

I'm using Go 1.17.1.

According to [Go 1.17 release note](https://golang.org/doc/go1.17#gofmt), if a file only has `// +build lines`, `//go:build` lines will be added.

> gofmt (and go fmt) now synchronizes //go:build lines with // +build lines. If a file only has // +build lines, they will be moved to the appropriate location in the file, and matching //go:build lines will be added. Otherwise, // +build lines will be overwritten based on any existing //go:build lines. For more information, see https://golang.org/design/draft-gobuild.

Should I remove `//go:build !ignore_autogenerated` ? @andreyvelich 
If we should remove it, it might be better to change the required Go version in [the developer guide](https://github.com/kubeflow/katib/blob/master/docs/developer-guide.md#requirements).

Reply:
Makes sense, let's remove it for now.

We are using Go 1.15 currently: https://github.com/kubeflow/katib/blob/master/go.mod#L3.
I remember that we had issues with Go 1.16 and our packages.
I will submit PR soon to use Go 1.17.

Reply:
> I will submit PR soon to use Go 1.17.

It sounds good!
I'd like to remove it.